yarn.lock

```diff
@@ -7296,7 +7288,7 @@ react-native-device-info@^10.6.0:
   resolved "https://registry.yarnpkg.com/react-native-device-info/-/react-native-device-info-10.6.0.tgz#51f1b2ec98abc32747149b0a5b7fb662b44d50e4"
   integrity sha512-/MmINdojWdw2/9rwYpH/dX+1gFP0o78p8yYPjwxiPhoySSL2rZaNi+Mq9VwC+zFi/yQmJUvHntkKSw2KUc7rFw==
 
-react-native-file-access@^2.4.4, react-native-file-access@^2.5.2:
+react-native-file-access@^2.5.2:
```

Comment:
Please, notify to Natalia of possible regressions related to changed library versions